include/bluetooth/bluetooth.h

```diff
@@ -277,6 +277,16 @@ enum {
 	 *  bt_conn_create_slave_le().
 	 */
 	BT_LE_ADV_OPT_DIR_MODE_LOW_DUTY = BIT(4),
+
+	/** Enable use of Private Resolvable Address in directed advertisements
```

Comment:
*Resolvable Private Address (RPA)
Please specify that this only affects InitA/TgtA and *not* AdvA